src/worker/shutdown.sh

```diff
@@ -1,6 +1,14 @@
-#!/bin/bash/
+#!/bin/bash
 
-arch=$1
+arch="nvidia"
+
+if [ -e '/dev/nvidiactl' ]; then
+    arch="nvidia"
+elif [ -e '/dev/kfd' ];then
+    arch="amd"
+else
+	arch='cpu'
```

Comment:
This isn't related to any changes in this PR, but can we kill the amd exporter in a way that doesn't assume a full path "/opt/rocm/rdc/bin/rdcd" (I know this is the default install location).